File: ip-messaging-demo-android/src/main/java/com/twilio/ipmessaging/demo/MessageTextView.java
package com.twilio.ipmessaging.demo;

import android.content.Context;
import android.graphics.Typeface;
import android.util.AttributeSet;
import android.widget.TextView;

public class MessageTextView extends TextView
{
    private static final String androidNS = "http://schemas.android.com/apk/res/android";

    public MessageTextView(Context context, AttributeSet attrs)
    {
        super(context, attrs);
        String font = attrs.getAttributeValue(androidNS, "fontFamily");
        if (font == null) {
            font = "AlegreyaSans-Light";
        }
        this.setTypeface(Typeface.createFromAsset(context.getAssets(), "fonts/" + font + ".ttf"));
    }
}

File: chat-demo-android/src/main/kotlin/com/twilio/chat/demo/views/MessageTextView.kt
package com.twilio.chat.demo.views

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet
import android.widget.TextView

class MessageTextView(context: Context, attrs: AttributeSet) : TextView(context, attrs) {

    init {
        var font: String? = attrs.getAttributeValue(androidNS, "fontFamily")
        if (font == null) {
            font = "AlegreyaSans-Light"
        }
        this.typeface = Typeface.createFromAsset(context.assets, "fonts/$font.ttf")
    }

    companion object {
        private val androidNS = "http://schemas.android.com/apk/res/android"
    }
}
